packages/server/.env.example

```
@@ -1,15 +1,18 @@
 # Solana
 RPC_URL=https://devnet.helius-rpc.com/?api-key=YOUR_HELIUS_API_KEY
-PROGRAM_ID=EXsr2Tfz8ntWYP3vgCStdknFBoafvJQugJKAh4nFdo8f
-CRANK_KEYPAIR=  # base58 or JSON array
+FALLBACK_RPC_URL=https://api.devnet.solana.com
+PROGRAM_ID=8n1YAoHzZAAz2JkgASr7Yk9dokptDa9VzjbsRadu3MhL
+ALL_PROGRAM_IDS=  # comma-separated (optional, defaults to known deployments)
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)